File: src/FizzBuzz.cpp
// FizzBuzz.cpp

#include <iostream>

void runFizzBuzz(int n) {
    for (int i = 1; i <= n; ++i) {
        if (i % 3 == 0 or i % 5 == 0) {
            if (i % 3 == 0) {
                std::cout << "Fizz";
            }
            if (i % 5 == 0) {
                std::cout << "Buzz";
            }
        } else {
            std::cout << i;
        }
        std::cout << std::endl;
    }
}

int main() {
    std::cout << "It's time for FizzBuzz!" << std::endl;
    int n;
    std::cout << "Please enter a positive integer (n > 0): ";
    std::cin >> n;
    std::cout << "n = " << n << std::endl;
    runFizzBuzz(n);
    return 0;
}

File: python/FizzBuzz.py
# FizzBuzz.py

def main():
    print("It's time for FizzBuzz!")
    n = int(input("Please enter a positive integer (n > 0): "))
    print(f"n = {n}")
    runFizzBuzz(n)

def runFizzBuzz(n):
    for i in range(1, n+1):
        message = ""
        if i % 3 == 0:
            message += "Fizz"
        if i % 5 == 0:
            message += "Buzz"
        if not message:
            message = str(i)
        print(message)

if __name__ == "__main__":
    main()
